swigwriter.vala

```diff
@@ -248,7 +248,7 @@ public class SwigWriter : ValabindWriter {
 			extends += "%%extend %s {\n".printf (name);
 
 			if (freefun != null && freefun != "")
-				extends += "\t~%s() {\n\t\t%s(self);\n\t}\n".printf (name, freefun);
+				extends += "\t~%s() {\n\n\t}\n".printf (name); // we do not need a free, cause it will be done in the lib.
```

Comment:
i think this should be fixed in RList, SdbList or the APIs or VAPIs, but not this way because that introduces a memory leak. can you provide some examples of scripts that produce this error?